src/python/test/test_subsampling.py

```diff
@@ -61,7 +61,7 @@ def test_simple_choose_n_farthest_points_randomed():
 
     # Go further than point set on purpose
     for iter in range(1, 10):
-        sub_set = gudhi.choose_n_farthest_points(points=point_set, nb_points=iter)
+        sub_set = gudhi.choose_n_farthest_points(points=point_set, nb_points=iter, fast=False)
```

Comment:
`fast=True/False` mechanism is not really tested but it would imply a `for fast in [True, False]:` on each test, and it is not really convenient. Maybe something smarter could be done.

Reply:
I did the `for` thing for now, plus one less trivial check that both versions return the same thing.